lib/components/arrowsNavigationWrap/arrowsNavigationWrap.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ArrowsNavigationWrap = ({ children, tabbingSelector, parentSelector }) => {
+  let parent = document.querySelector(parentSelector);
+
+  React.useLayoutEffect(() => {
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    if (parent === null) parent = document.querySelector(parentSelector);
+
+    const arrowsNavigation = (event) => {
+      const items = parent.querySelectorAll(tabbingSelector);
```

Comment:
`parent` should not be necessary. We should always be querying within this component.